scripts/split-e2e-tests.ts

```diff
@@ -3,6 +3,11 @@ import * as glob from 'glob';
 import * as fs from 'fs-extra';
 import * as yaml from 'js-yaml';
 
+const REPO_ROOT = join(__dirname, '..');
+const SUPPORTED_REGIONS_PATH = join(REPO_ROOT, 'scripts', 'e2e-test-regions.json');
+const AMPLIFY_SUPPORTED_REGIONS: string[] = JSON.parse(fs.readFileSync(SUPPORTED_REGIONS_PATH, 'utf-8'));
+const AWS_REGIONS_TO_RUN_TESTS = AMPLIFY_SUPPORTED_REGIONS;
+const SUPPORTED_OPT_IN_REGIONS = ['ap-east-1', 'eu-south-1', 'me-south-1'];
```

Comment:
Let's include the opt in status in the region list, rather than re-typing a list of regions. Basically, we should reuse the structure of the [`Account.ListRegions` API](https://docs.aws.amazon.com/accounts/latest/reference/API_ListRegions.html) `Regions` response element.